api/src/main/java/edu/cornell/library/scholars/webapp/controller/api/distribute/rdf/graphbuilder/ThreadsafeGraphBuilder.java

```diff
@@ -0,0 +1,13 @@
+/* $This file is distributed under the terms of the license in /doc/license.txt$ */
+
+package edu.cornell.library.scholars.webapp.controller.api.distribute.rdf.graphbuilder;
+
+/**
+ * TODO
+ * 
+ * Marker interface asserting that the GraphBuilder may be called by several
+ * threads at once without conflict
+ */
+public interface ThreadsafeGraphBuilder {
```

Comment:
Also, probably should be removed (or used in the implementation of ParallelGraphBuilder class)

Reply:
Removed